controllers/badge.js

```diff
@@ -20,6 +21,29 @@ const getBadges = async (req, res) => {
   }
 }
 
+const getUserBadges = async (req, res) => {
+  try {
+    const result = await badgeQuery.fetchUserBadges(req.params.username)
+    if (result.userExists) {
+      if (result.userBadges.length === 0) {
+        return res.json({
+          message: 'This user does not have any badges'
+        })
```

Comment:
This code is looking necessarily nested and complex. Can we simply return an empty array (which will be implicit, we don't have to manually do anything there) and custom message, in case of 0 badges?
We can use ternary operator, so we'll be having better readability. What do you say? You can check how we're doing that in other controllers

Reply:
Yes let's go with the ternary operator, I'll make the changes.